docs/docs/cmd/viva/engage/engage-community-add.mdx

```diff
@@ -0,0 +1,170 @@
+    "createdDateTime": "2024-04-10T12:16:18.0240578Z",
+    "lastActionDateTime": "2024-04-10T12:16:18.0240592Z",
+    "status": "succeeded",
+    "statusDetail": null,
+    "resourceLocation": "https://graph.microsoft.com/beta/employeeExperience/communities('eyJfdHlwZSI6Ikdyb3VwIiwiaWQiOiIxODQ4MTMxNTAyMDgifQ')",
+    "operationType": "createCommunity",
+    "resourceId": "eyJfdHlwZSI6Ikdyb3VwIiwiaWQiOiIxODQ4MTMxNTAyMDgifQ"
+  }
+  ```
+
+  </TabItem>
+  <TabItem value="Text">
+
+  ```text
+  createdDateTime   : 2024-04-10T12:45:03.4020639Z
+  id                : eyJfdHlwZSI6IkxvbmdSdW5uaW5nT3BlcmF0aW9uIiwiaWQiOiI5NzRkMjUzZS1jMGUzLTQ2YjgtYTkzMy0zZDZhODU1NGE0OWUiLCJvcGVyYXRpb24iOiJDcmVhdGVDb21tdW5pdHkifQ
+  lastActionDateTime: 2024-04-10T12:45:03.4020656Z
+  operationType     : createCommunity
+  resourceId        : eyJfdHlwZSI6Ikdyb3VwIiwiaWQiOiIxODQ4MTUwNTA3NTIifQ
+  resourceLocation  : https://graph.microsoft.com/beta/employeeExperience/communities('eyJfdHlwZSI6Ikdyb3VwIiwiaWQiOiIxODQ4MTUwNTA3NTIifQ')
+  status            : succeeded
+  statusDetail      : null
+  ```
+
+  </TabItem>
+  <TabItem value="CSV">
+
+  ```csv
+  id,createdDateTime,lastActionDateTime,status,statusDetail,resourceLocation,operationType,resourceId
+  eyJfdHlwZSI6IkxvbmdSdW5uaW5nT3BlcmF0aW9uIiwiaWQiOiJiMDU0MjFmNC0zNzU5LTQ3NzgtOTVhYi01NzhiYjAzZGNiMWQiLCJvcGVyYXRpb24iOiJDcmVhdGVDb21tdW5pdHkifQ,2024-04-12T11:24:29.074105Z,2024-04-12T11:24:29.0741064Z,succeeded,,https://graph.microsoft.com/beta/employeeExperience/communities('eyJfdHlwZSI6Ikdyb3VwIiwiaWQiOiIxODUwMTc2NzE2ODAifQ'),createCommunity,eyJfdHlwZSI6Ikdyb3VwIiwiaWQiOiIxODUwMTc2NzE2ODAifQ
+  ```
+
+  </TabItem>
+  <TabItem value="Markdown">
+
+  ```md
+  # viva engage community add --displayName "Software engineers" --description "A community for all software engineers" --privacy "private" --wait "true"
+
+  Date: 10/04/2024
+
+  ## eyJfdHlwZSI6IkxvbmdSdW5uaW5nT3BlcmF0aW9uIiwiaWQiOiI5ZTgzMDRjMC1mZmY0LTRkODgtYTM4OS1mNDY0MmQ3OGZlMTciLCJvcGVyYXRpb24iOiJDcmVhdGVDb21tdW5pdHkifQ
+
+  Property | Value
+  ---------|-------
+  id | eyJfdHlwZSI6IkxvbmdSdW5uaW5nT3BlcmF0aW9uIiwiaWQiOiI5ZTgzMDRjMC1mZmY0LTRkODgtYTM4OS1mNDY0MmQ3OGZlMTciLCJvcGVyYXRpb24iOiJDcmVhdGVDb21tdW5pdHkifQ
+  createdDateTime | 2024-04-10T12:47:47.3488087Z
+  lastActionDateTime | 2024-04-10T12:47:47.3488107Z
+  status | succeeded
+  resourceLocation | https://graph.microsoft.com/beta/employeeExperience/communities('eyJfdHlwZSI6Ikdyb3VwIiwiaWQiOiIxODQ4MTUyMzkxNjgifQ')
+  operationType | createCommunity
+  resourceId | eyJfdHlwZSI6Ikdyb3VwIiwiaWQiOiIxODQ4MTUyMzkxNjgifQ
+  ```
+
+  </TabItem>
+</Tabs>
+
+### Without --wait:
+
+<Tabs>
+  <TabItem value="JSON">
+
+  ```json
+  https://graph.microsoft.com/beta/employeeExperience/engagementAsyncOperations('eyJfdHlwZSI6IkxvbmdSdW5uaW5nT3BlcmF0aW9uIiwiaWQiOiI4ZmM2NzEyZS0wMWY4LTQxN2YtYWNmMS1iZTJiYmMxY2FjNGQiLCJvcGVyYXRpb24iOiJDcmVhdGVDb21tdW5pdHkifQ')
```

Comment:
When output is JSON, this should be a JSON string. So we're missing quotes around this string.